catalog/task/run-file-updates/0.2/run-file-updates.yaml

```diff
@@ -32,10 +32,10 @@ spec:
       description: Workspace where the file updates to apply are defined
   steps:
     - name: run-script
-      image: quay.io/hacbs-release/release-utils:34901b1694963265ac2dcfecc2f56e15a2a8c7fe
+      image: quay.io/hacbs-release/release-utils@sha256:acf0a7c27fc3770eaa26098a8404efd343550e83b760dc6f8261355a73aed549
```

Comment:
We've been using the image tags (generated from commit sha) everywhere. Can we stick to that to be consistent?

Reply:
fixed.